tests/gocase/unit/type/stream/stream_test.go

```diff
@@ -2083,7 +2086,7 @@ func TestStreamOffset(t *testing.T) {
 			Count:     1,
 			Lower:     "1-0",
 			Higher:    "1-0",
-			Consumers: map[string]int64{"myconsumer": 1},
+			Consumers: map[string]int64{consumerName: 1},
```

Comment:
Can we extract the `consumerName` to a variable like `streamName` and `groupName`?